src/driver/franka_plan_runner.h

```diff
@@ -240,8 +249,11 @@ class FrankaPlanRunner {
     plan_.reset();
     plan_utime_ = -1;
     plan_start_utime_ = -1;
+    franka_time_ = 0;
   }
 
+  bool IsContinuousWithCurrentPlan(const std::unique_ptr<PPType>& plan);
```

Comment:
need docstring. _How_ does it test if the function is continuous with the current plan?

Reply:
how is this different from`IsStartFarFromCurrentJointPosition`? aren't they checking the same thing?